### ISSUE TITLE
   replaced texture Game will Crash 

### ISSUE THREAD

**Issue by Coinswang** [nesrak1/UABEA#183]
UnityVersion 2020.3.29f1c1
I replace texture  like 'LiberationSans SDF Atlas' game will crash
Error :/base.apk/assets/bin/Data/resources.assets' is corrupted! Remove it and launch unity again!

game link:
https://play.google.com/store/apps/details?id=com.together.SurvivalGameHost

**Comment by nesrak1** [comment]
Did you use the texture edit plugin? Can you export the original texture and reimport it without issues?

**Comment by Coinswang** [comment]
yes ,I use the  texture edit plugin,i export the texture don't change anything and reimport it, the game will crash too.
and i use beyond Hex Mode see two files, the new file have many changes compared with the old file

**Comment by Shua-github** [comment]
You can try using the folder comparison tool to see if there are any extra files (I worked on the game I modified)
